loader/loader.go

```diff
@@ -173,15 +177,20 @@ syntax = "proto3";
 	}
 	defer os.Remove("gunk-proto")
 
+	debug.PrintCommand("protoc", "-o/dev/stdout", "--include_imports", "gunk-proto")
```

Comment:
Something that I don't like that much about this approach is how we need to:

* Remember to put a `debug.PrintCommand` next to every `exec.Command`
* Keep the two string lists in sync

How about adding a wrapper for `exec.Command`, which would:

* log the command to run, if necessary
* use `exec.Command`
* redirect its stderr to `os.Stdout`, if necessary

Then, if a user needs to modify the `exec.Cmd` such as setting `Stdin`, they can still do that.

As for the name, `debug.ExecCommand` is the best I can come up with. I think there's a better name than `debug` for this, since it's more than just debugging. Regular users could also benefit, so perhaps `log` would be a better package name.

Reply:
My suggestion above would keep everything working, with the caveat that the command's stderr would be streamed to the console, as opposed to buffered and then printed all at once. But that seems fine - better, even - to me.

Reply:
This should be done (if I understood what you were suggesting correctly)